src/redux/action-creator/adminList.ts

```diff
@@ -51,35 +51,58 @@ export const fetchAdminList = () => {
     };
 };
 
-export const deleteAdminAction = (adminAddress: string) => {
+export const deleteAdminAction = (adminAddress: string, stargateClient: SigningStargateClient) => {
     return async (dispatch: Dispatch<AdminActions>): Promise<void> => {
         try {
             const kepler = await getKeplr();
             if (kepler) {
                 const sender = await getWalletAddress(kepler);
+
                 const msgDeleteAdminRequest = {
                     admin: adminAddress,
-                    requester: sender
+                    creator: sender
+                };
+                const msgAny = {
+                    typeUrl: "/cosmos.adminmodule.adminmodule.MsgDeleteAdmin",
+                    value: msgDeleteAdminRequest
                 };
+
+                const fee = {
+                    amount: coins(0, chainInfo.stakeCurrency.coinMinimalDenom),
+                    gas: "2000000"
+                };
+
                 console.log("sending", msgDeleteAdminRequest);
-                // TODO: add deletion request + fetch new admins
-                const admins = MOCK_ADMINS.filter((adm) => adm !== adminAddress);
-                MOCK_ADMINS = admins;
-                dispatch({ type: AdminListActionTypes.SET_LIST, payload: { admins } });
+                dispatch({ type: AdminListActionTypes.SET_LOADING, payload: { loading: true } });
+                const broadcastRes = await stargateClient.signAndBroadcast(sender, [msgAny], fee);
+
+                console.log("broadcast res", broadcastRes);
+
+                // @ts-ignore
+                if (broadcastRes?.code == 0) {
+                    const adminsResp: { admins: string[] } = await lcdClient.get(
```

Comment:
Why not use dispatch (fetchAdminList ())?

Reply:
dispatch cannot be used here with action creator